src/main/webapp/styles/zoom.css

```diff
@@ -0,0 +1,43 @@
+/* Zoom styling */
```

Comment:
Might be nice to make this a general styling sheet for all tech page styles like Zoom, Facebook, Google meet. just so we dont need a separate file for each tech we add.

Reply:
I kept it separate for now just in case we choose different formatting for other pages, but yes in general once I add another tutorial, I will most likely have one styling sheet.

Reply:
Yes I agree changed names to reflect that. I had it like this in case we decided that the formatting of other pages should be different.